src/ensembl/webpack/webpack.config.prod.js

```diff
@@ -103,7 +103,12 @@ const plugins = [
   new WorkboxPlugin.GenerateSW({
     swDest: '../service-worker.js', // save service worker in the root folder (/dist) instead of /dist/static
     clientsClaim: true,
-    skipWaiting: true
+    skipWaiting: true,
+    exclude: [/index.html/],
+    runtimeCaching: [{
+      urlPattern: ({ event }) => event.request.mode === 'navigate',
+      handler: 'NetworkOnly'
```

Comment:
I think what we need here is `NetworkFirst` as `NetworkOnly` will cause the site to be completely down during the build time.

Reply:
Two things here:

1) Our site will **not** be down during build time, because while the build is happening our site is still served by the container with the previous build.
2) I can replace NetworkOnly with NetworkFirst in hope that some of our pages may be operational offline (although I doubt any of them will be useful offline).

**TLDR:** NetworkFirst probably wouldn't be worse than NetworkOnly and might in some cases be slightly better.

Reply:
> Our site will not be down during build time, because while the build is happening our site is still served by the container with the previous build.

Cool, that makes sense. Are we sure that `index.html` will always be available on the network during the deployment?

Since we exclude `index.html` from getting cached, if at some point it is not available (for a fraction of time), the handler is going to fallback and look for it in the cache where it won't exist.

Reply:
> Are we sure that index.html will always be available on the network during the deployment?

That's up to Kamal to say; but if there are more than one containers with nginx (which there probably should be), then there shouldn't be a situation when index.html (or any static files) become unavailable during deployment.

Reply:
Hi @kamaldodiya, any thoughts on this?